src/test/ui/resolve/suggest-path-instead-of-mod-dot-item.stderr

```diff
@@ -2,55 +2,67 @@ error[E0423]: expected value, found module `a`
   --> $DIR/suggest-path-instead-of-mod-dot-item.rs:27:5
    |
 27 |     a.I
-   |     ^ did you mean `a::I`?
+   |     ^^^
+   |     |
+   |     did you mean `a::I`
```

Comment:
This might be caused by the same span being used as primary and secondary. The primary span didn't have any text, but the secondary did. In terminal output the `^^^` would be red and the `did you  (...)` would be blue. (This is an educated guess.)